modules/eco-self-node-remediation-operator-installation-cli.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-self-node-remediation-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-self-node-remediation-operator-using-cli_{context}"]
+= Installing the Self Node Remediation Operator by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the Self Node Remediation Operator.
+
+You can install the Self Node Remediation Operator in your own namespace or in the `openshift-operators` namespace.
+
+To install the Operator in your own namespace, follow the steps in the procedure.
+
+To install the Operator in the `openshift-operators` namespace, skip to step 3 of the procedure because the steps to create a new `Namespace` custom resource (CR) and an `OperatorGroup` CR are not required.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the Self Node Remediation Operator:
+.. Define the `Namespace` CR and save the YAML file, for example, `self-node-remediation-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: self-node-remediation
+----
+.. To create the `Namespace` CR, run the following command:
++
+[source,terminal]
+----
+$ oc create -f self-node-remediation-namespace.yaml
+----
+
+. Create an `OperatorGroup` CR:
+.. Define the `OperatorGroup` CR and save the YAML file, for example, `self-node-remediation-operator-group.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: self-node-remediation-operator
+  namespace: self-node-remediation
+----
+.. To create the `OperatorGroup` CR, run the following command:
++
+[source,terminal]
+----
+$ oc create -f self-node-remediation-operator-group.yaml
+----
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `self-node-remediation-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: self-node-remediation-operator
+    namespace: self-node-remediation <1>
+spec:
+    channel: stable
+    installPlanApproval: Manual <2>
+    name: self-node-remediation-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+    package: self-node-remediation-operator
```

Comment:
I think package name is `self-node-remediation` without `operator`

@mshitrit  Can you confirm this?

```
[kni@provisionhost-0-0 ~]$ oc get sub -A
NAMESPACE             NAME                    PACKAGE                 SOURCE          CHANNEL
openshift-operators   self-node-remediation   self-node-remediation   snr-manifests   stable
```

Reply:
Yep :+1: